app/upgrades/v13/constants.go

```diff
@@ -13,7 +15,7 @@ var Upgrade = upgrades.Upgrade{
 	UpgradeName:          UpgradeName,
 	CreateUpgradeHandler: CreateUpgradeHandler,
 	StoreUpgrades: store.StoreUpgrades{
-		Added:   []string{valsetpreftypes.StoreKey},
+		Added:   []string{valsetpreftypes.StoreKey, swaproutertypes.StoreKey},
```

Comment:
Discussed offline - going to wait for the v14 upgrade handler to be auto-generated with https://github.com/osmosis-labs/osmosis/pull/3100/files

Then, port this logic there.

It shouldn't be a problem for now since this is not going to be backported to `v13.x`, and we can keep e2e testing the upgrade to v13 untile the v14 upgrade handler is created.